src/HelpText.cpp

```diff
@@ -239,18 +239,18 @@ static wxString HelpTextBuiltIn( const wxString & Key )
 // DA: Support methods text.
 #ifdef EXPERIMENTAL_DA
          // Deliberately not translated.
-         << wxT("<center><h3>DarkSneedacity ")
+         << wxT("<center><h3>DarkAudacity ")
```

Comment:
Yes, reverting this one as well for now while the rework of theming is going on. This code won't appear anyway in normal circumstances.

Reply:
Why does it need to be reverted? Didn't we recursively sneedfiy the whole src folder?

Reply:
Because we don't have a "DarkSneedacity" yet AFAIK, so I don't want the code to look misleading. However it might be better to remove all this DarkAudacity ifdef anyway, but haven't tested yet whether that would break something.

Reply:
The dark theme already works, if that's what you mean.

Reply:
it was already found the theming was awful and dark~~aud~~sneedacity was a workaround for it, I'll fish it out in another pr, will force this one to be rebased